templates/catalog/catalog.html

```diff
@@ -7,16 +7,21 @@
   {% breadcrumbs_with_siblings page %}
   <h1>{{ page.display.h1 }}</h1>
 
-  <ul class="catalog-list">
-    {% recursetree nodes %}
-      <li class="catalog-list-item">
-        <a href="{{ node.url }}" class="catalog-list-item">{{ node.name }}</a>
-        {% if not node.is_leaf_node %}
-          <ul class="catalog-list-children">
-            {{ children }}
+  <div class="container container-fluid">
+    <div class="row around-xs">
+      {% for root_name, children in matrix.items %}
+        <div class="category-block">
+          <h2>{{ root_name }}</h2>
+          <ul>
+            {% for category in children %}
```

Comment:
```suggestion
            {% for child in children %}
```
up to you

Reply:
we already use category in the same place at STB. So, i'll leave like this for consistency.
`category` name is relevant too and has only scope size in 3 lines